### PR TITLE
refactor: #id 19713 rename "my apps" to "advanced app launcher"

### PR DESCRIPTION
**Pull request by zona-ciq**
refactor: #id 19713

[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/CARD/details)

**Description of change**
* rename "my apps" to "advanced app launcher" everywhere in the application except for the UI's foldername. This includes instances in configuration and directory structure.
* All Instances of "My Apps" is now "Advanced App Launcher". All Instances of "myApps" is now "advancedAppLauncher" (or whatever engineering suggests)

**Description of testing**
1. Test this branch along with `19713-advanced-app-launcher` branch for finsemble core
1. Build and start finsemble *fresh*, make sure app launcher is working as expected and all instances of "my apps" is changed to "advanced app launcher"

### CODE REVIEW

**Comment by amaroks** [comment]
@zona-ciq you got some conflicts here :)

File: src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx [hunk 1]
```diff
@@ -214,10 +214,16 @@ export default class FoldersList extends React.Component {
 
 		const EDITABLE_FOLDER_ICON_CLASS = 'ff-adp-hamburger'
 
-		let nameField = folder.icon === EDITABLE_FOLDER_ICON_CLASS && this.state.renamingFolder === folderName ?
-			<input id="rename" value={this.state.folderNameInput}
-				onChange={this.changeFolderName}
-				onKeyPress={this.keyPressed} className={this.state.isNameError ? "error" : ""} autoFocus /> : folderName;
+		let nameField;
```
Comment (d4hines) [inline src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx]: It looks like you've changed the logic here by adding a case: `folderName === "Advanced App Launcher"`. Can you explain this change?

Comment (zona-ciq) [reply src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx]: I confirmed this with Josh and Brad: we want the UI to show "App Launcher" even if users are using advanced app launcher

Comment (d4hines) [reply src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx]: Ok - makes sense then.

**Pull request merged**